src/wasm-interpreter.h

```diff
@@ -1252,7 +1252,7 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
     // Check if a constant value has been set in the context of this runner.
     auto iter = localValues.find(curr->index);
     if (iter != localValues.end()) {
-      return Flow(std::move(iter->second));
+      return Flow(iter->second);
```

Comment:
This one was obviously wrong, since `localValues`, a `std::unordered_map`, contents can change in various ways.

Reply:
Hmm yeah I have no idea what happens if you move out of an iterator 🤷‍♂️ And of course in C++ that means it should be avoided like a pit of snakes.